src/components/FloatingActionButton.tsx

```diff
@@ -122,9 +130,8 @@ const FloatingActionButton = React.forwardRef(({onPress, isActive, accessibility
             </View>
         </Tooltip>
     );
-});
+}
 
-FloatingActionButton.propTypes = propTypes;
 FloatingActionButton.displayName = 'FloatingActionButton';
 
 export default FloatingActionButton;
```

Comment:
<img width="616" alt="Screenshot 2024-01-09 at 3 12 01 PM" src="https://github.com/Expensify/App/assets/108292595/e6ff3508-bbb2-46cd-8715-b4312e0d299e">


This PR caused regression. Seems like you forgot to wrap with `forwardRef` here

```suggestion
export default forwardRef(FloatingActionButton);
```

@JKobrynski can you raise quick PR?

Reply:
Now this caused a few deploy blockers - #34185, #34183
I will go ahead and raise quick PR.